pkg/util/iptables/iptables.go

```diff
@@ -449,16 +449,6 @@ func getIptablesVersionString(exec utilexec.Interface) (string, error) {
 	return string(bytes), nil
```

Comment:
The old version of this called extractIptablesVersion() but you dropped that.  I patched this into my build and it fails to parse the iptables version.